_extras/guide.md

```diff
@@ -1,6 +1,25 @@
 ---
+layout: page
 title: "Instructor Notes"
 ---
-FIXME
 
-{% include links.md %}
+## Resouces for Instructors
+
+## Workshop Structure
+
+[Instructors, please add notes on your experience with the workshop structure here.]
+
+## Technical tips and tricks
+
+If using an HPC with this workshop, it is recommended that workshop organizers make sure that attendees have access to the HPCs before the workshop
```

Comment:
I think we would strongly recommend using an HPC platform for providing Singularity training since I suspect that is likely to be the environment in which most users will want to use Singularity.

This has been suggested in the installation section below but I think I'd be inclined to say that here too - I'm happy to make that update.